StyleCop.Analyzers/StyleCop.Analyzers/Helpers/IndentationHelper.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace StyleCop.Analyzers.Helpers
+{
+    using System.Linq;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.CSharp;
+    /// <summary>
+    /// Provides helper methods to work with indentation.
+    /// </summary>
+    internal static class IndentationHelper
+    {
+        /// <summary>
```

Comment:
There are spacing missing between all elements in class. This will trigger SA1516.